docs/serverless/ingest/auto-import.mdx

```diff
@@ -0,0 +1,85 @@
+
+While Elastic has 400+ [prebuilt data integrations](((integrations-docs))), Automatic Import helps you extend data coverage to other security-relevant technologies and applications. Elastic integrations (including those created by Automatic Import) normalize data to [Elastic Common Schema (ECS)](https://www.elastic.co/blog/ecs-elastic-common-schema-otel-opentelemetry-faq), which creates uniformity across dashboards, search, alerts, machine learning, and more. 
+
+
+<DocCallOut title="Tip">
+Click [here](https://elastic.navattic.com/automatic-import) to watch a demo of how Automatic Import works before setting it up yourself.
+</DocCallOut>
+
+<DocCallOut title="Requirements">
+
+- A working <DocLink slug="/serverless/security/connect-to-bedrock" text="Amazon Bedrock connector"/>. Automatic Import currently works with all variants of Claude 3. In this technical preview, other models are not yet supported.
+- A [security analytics complete](https://www.elastic.co/pricing/serverless-security) subscription.
+- A sample of the data you want to import in JSON or NDJSON format. 
+
+</DocCallOut>
+
+
+## Create a new custom integration
+
+1. In your ((elastic-sec)) deployment, click **Add integrations**.
+2. Under **Can't find an integration?** click **Create new integration**.
+
+<DocImage url="images/auto-import-create-new-integration-button.png" alt="The Integrations page with the Create new integration button highlighted" />
+
+3. Click **Create integration**.
+4. Select an <DocLink slug="/serverless/security/connect-to-bedrock" text="Amazon Bedrock connector"/>. 
+5. Define how your new integration will appear on the integrations page by providing a **Title**, **Description**, and **Logo**.  Click **Next**.
+6. Define your integration's package name, which will prefix the imported event fields. 
+7. Define your **Data stream title**, **Data stream description**, and **Data stream name**. These fields will appear on the integration's configuration page to help identify the data stream it writes to.
+8. Select your [**Data collection method**](https://www.elastic.co/guide/en/beats/filebeat/current/configuration-filebeat-options.html). This determines how your new integration will ingest the data, for example from an S3 bucket, an HTTP endpoint, or a file stream.
+9. Upload a sample of your data in JSON format. Make sure to include all the types of events that you want the new integration to handle. 
+<DocCallOut title="Best practices for sample data"> 
```

Comment:
Nice use of a custom callout! It'd be nice to indent this under the step, for a slightly different visual flow, but I wouldn't wrestle with the MDX more than 5 seconds to try to do this. Definitely optional.